packages/discovery-provider/src/api/v1/helpers.py

```diff
@@ -591,6 +592,23 @@ def extend_purchase(purchase):
     return new_purchase
 
 
+def extend_feed_item(item):
+    if item.get("track_id"):
+        return {
+            "type": "track",
+            "item": extend_track(item),
+        }
+    if item.get("playlist_id"):
+        extended_playlist = extend_playlist(item)
+        # TODO: https://linear.app/audius/issue/PAY-3398/fix-playlist-contents-serialization
+        extended_playlist["playlist_contents"] = extended_playlist["added_timestamps"]
```

Comment:
ahhhh another one 😖

Reply:
Don't worry, I added a ticket with a plan to address it 😁